### ISSUE TITLE
[Dev Asks] Can someone send me their player save?

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5512]
So I lost my survival character save when I pushed the last build as I changed the planet radius (so all my stuff is now floating in orbit).
Can someone send me its survival save so I can test all your issues?
I plan on starting a new one again to test more stuff but it takes a few hours to get there.

Cheers

**Comment by Huttrem** [comment]
@tsunamayo i send you my in the other post about the trasparent tank

**Comment by asanagisae** [comment]
Unfortunately my save is past the Github limit of 25 MB, even when zipped. In cases like this, would it be safe to delete the data in the "Terrain" folder?

**Comment by Xerma** [comment]
[Burma Survival.zip](https://github.com/tsunamayo/Starship-EVO/files/14274099/Burma.Survival.zip)

**Comment by tsunamayo** [comment]
> Unfortunately my save is past the Github limit of 25 MB, even when zipped. In cases like this, would it be safe to delete the data in the "Terrain" folder?

Yes! How big is your terrain folder? This might also become an issue...

**Comment by asanagisae** [comment]
It's 54.8 MB currently and growing whenever I mine a new patch of ore.

I've had save files grow past the 1 GB point in space engineers. I think that's excessive, but maybe a couple hundred MB is ok for an active multiplayer world?

**Comment by Janssendel** [comment]
[212 - Copy.zip](https://github.com/tsunamayo/Starship-EVO/files/14274946/212.-.Copy.zip)
it has a large ship already so you can test whatever you want.

**Comment by Uncle-Ulty** [comment]
[Survival 2.zip](https://github.com/tsunamayo/Starship-EVO/files/14274998/Survival.2.zip)
